tests/network/bond/test_bond_modes.py

```
@@ -151,12 +160,18 @@ def bridge_on_bond_fail_over_mac(
 
 
 @pytest.fixture()
-def active_backup_bond_with_fail_over_mac(index_number, worker_node1, nodes_available_nics):
+def active_backup_bond_with_fail_over_mac(index_number, worker_node1, hosts_common_available_ports):
     bond_index = next(index_number)
+    if len(hosts_common_available_ports) < 2:
+        LOGGER.error(
+            f"Insufficient common NICs across worker nodes: found {len(hosts_common_available_ports)} "
+            f"({hosts_common_available_ports})"
+        )
+        raise AssertionError("At least 2 common NICs are required.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)